docs/OpenCLCOnVulkan.md

```diff
@@ -902,3 +904,47 @@ These extension built-in functions are not supported:
 - `sub_group_commit_write_pipe()`
 - `get_kernel_sub_group_count_for_ndrange()`
 - `get_kernel_max_sub_group_size_for_ndrange()`
+
+### Numerical Compliance
+
+Clspv is not able to reach full precision requirements on the supported builtin
+functions in all cases. Instead, currently, it is able to meet the requirements
+tested by CTS under relaxed precision requirements. In order to achieve this
```

Comment:
by OpenCL CTS...

And is that for only certain tested implementations, or do we know that in general?

Reply:
I added a note to describe that this has been tested across 5 implementations.